x-pack/plugins/observability_solution/logs_data_access/server/services/get_logs_rates_service/index.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from '@kbn/core/server';
+import { estypes } from '@elastic/elasticsearch';
+import { RegisterServicesParams } from '../register_services';
+
+export interface Params {
```

Comment:
**nit**: `Params` feels a bit generic for an interface that consumers of this service might require for typing on the usage site, can we improve this naming? Also, it's not exported by the plugin index, making it unreachable by other plugins unless they import `@kbn/logs-data-access-plugin/server/services/get_logs_rates_service`